com/win32comext/axdebug/Test/host.py

```diff
@@ -97,66 +101,19 @@ def GetPathName(self):
         try:
             return win32api.GetFullPathName(self.module.__file__), 1
         except (AttributeError, win32api.error):
-            raise Exception(scode == E_FAIL)
+            raise Exception(scode=winerror.E_FAIL)
 
     def GetFileName(self):
         # Result is a string with just the name of the document, no path information.
         trace("GetFileName")
-        return os.path.split(module.__file__)
+        return os.path.split(self.module.__file__)
 
     def NotifyChanged():
         trace("NotifyChanged")
         raise Exception(scode=winerror.E_NOTIMPL)
 
 
-def TestSmartHelper():
```

Comment:
These are commented out and contain a bunch of undefined names / unbound variables.
If you still think it's worth keeping around, I'll comment them out instead.